Extensions/Firebase/C_firebasetools/D_functionstools.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Firebase Tools Collection.
+ * @fileoverview
+ * @author arthuro555
+ */
+
+/**
+ * Firebase Functions Event Tools.
+ * @namespace
+ */
+gdjs.evtTools.firebase.functions = {
+  /**
+   * Call an http function.
+   * @param {string} httpFunctionName - The name of the function to call
+   * @param {string} [parameters] - Parameters for the function either as a JSON object or a string.
```

Comment:
"either as a JSON object or a string" => JSON **is** a string :)
Rather, say: "either as a JS object or a JSON string"

Also make sure your typings are up to date with your comments, here something like: `string | Object`